zsh/.zshrc

```diff
@@ -356,6 +356,13 @@ fi
 mkdir -p ~/.zshrc.pre-plugins.d
 load-shell-fragments ~/.zshrc.pre-plugins.d
 
+# Convert .zqs-additional-plugins to new format
+if [[ -f ~/.zqs-additional-plugins ]]; then
+  sed -e 's/^./zgenom load &/' ~/.zqs-additional-plugins > ~/.zqs-add-plugins
```

Comment:
Make this `>>` in case they already hand made a `~/.zqs-add-plugins` file.

Rather than have extra plugins loaded in multiple places, it is probably better to do a `mkdir -p ~/.zshrc.add-plugins.d` and append the plugins found to `~/zshrc.add-plugins.d/0000-transferred-plugins`.

Also move 359-364 ahead of 135 so that the extra plugins are transferred to `.zshrc-add-plugins.d` before that gets loaded.

Reply:
Great idea! Check my fixes

Reply:
Remove plugin loading from a separate file?

Reply:
Now there will be no separate file 👌🏻